### ISSUE TITLE
Possible scale issue

### ISSUE THREAD

**Issue by johnsone006** [BadDogSkyrim/PyNifly#69]
So when I used this addon to import a .nif file-specifically a vanilla floor mesh from SSE-I had issues with clipping. Also, upon further investigation, I learned that just one side of the floor mesh was 1680 ft long, which was waaaaaaaaaaaaay too big to be realistic, so I opened that mesh in the Skyrim Creation kit, and used my own method to measure the mesh. It turned out to be either 12 ft sq or 24 ft sq.
TLDR: I have reason to believe that the addon scales meshes waaaaaaaaaaaaaay up and then applies the transformations during the conversion process. I would like it to not do that.

**Comment by johnsone006** [comment]
SOVintFloor1

**Comment by BadDogSkyrim** [comment]
By "black preview screen" you mean the preview screen was normal but the object was black? Often that's because the mesh says it has vertex colors but it doesn't. BUT that shouldn't happen with my exporter because the flag is tied to the vertex colors. Then you say you got it to show properly, so is there a problem here?

**Comment by BadDogSkyrim** [comment]
Okay, that's weird unless whatever is making the render window black makes the preview black.

**Comment by johnsone006** [comment]
- [ ] - So I have been scaling the mesh down, and applying the transforms before exporting the mesh. But here's a weird thing: when I tried to reimport the mesh I just exported, blender opened a file of what looks an awful lot like an armature and not at all like a floor

**Comment by BadDogSkyrim** [comment]
I dunno. If you want to send me the .blend or the .nif I can have a look.

**Comment by johnsone006** [comment]
Okay, so .nif files are not compatible here. If you would like, I could email it to you.
Or you could join the discord that I've been active in. There are a lot of mod authors in there, as well as at least a few professionally trained IT pros. We're all working on creating a site to rival NexusMods, so we're looking for new recruits. :)

**Comment by johnsone006** [comment]
Update: one of my computer tech friends walked me through making it work for me.  Yay! I think it was because I had both the LE version of this plugin and the SSE installed on the same instance of Blender.
